app/static/css/style.css

```diff
@@ -72,19 +72,14 @@ div.header {
 
 .btn-primary {
   border: 2px solid var(--cirrus-primary) !important;
-  border-radius: 2px;
+  border-radius: 2px
```

Comment:
?

Reply:
I have no clue how that happened LOL ill fix it rn